### ISSUE TITLE
The problem about releative path

### ISSUE THREAD

**Issue by ejzhang** [gogits/gogs#4260]
The issue will be closed without any reasons if it does not satisfy any of following requirements:

1. Please speak English, we have forum in [Chinese](https://discuss.gogs.io/c/getting-help/getting-help-chinese).
2. Please post questions or config/deploy problems on our forum: https://discuss.gogs.io, here are bugs and feature requests only.
3. Please take a moment to search that an issue doesn't already exist.
4. Please give all relevant information below for bug reports; incomplete details considered invalid report.

**You MUST delete above content including this line before posting; too lazy to take this action considered invalid report.**

- Gogs version (or commit ref): 
- Git version: 0.10.8
- Operating system: Windows 10
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [√] SQLite
- Can you reproduce the bug at https://try.gogs.io:
  - [ ] Yes (provide example URL)
  - [√] No
  - [ ] Not relevant
- Log gist (usually found in `log/gogs.log`):

## Description

Use SQLite3 and save relative path "data/gogs.db" in app.ini, running as windows service, the database file "gogs.db" was saved in "C:\Windows\System32\data\".
I think all of the configuration, database, log, temporary files, if they didn't write in the configuration or use relative path should be in the Gogs root directory default. It should be unified, all the relative path add a prefix of the Gogs root directory.

**Comment by Unknwon** [comment]
Hi, please read post template.

**Comment by ejzhang** [comment]
Sorry, my english is so poor!

**Comment by Unknwon** [comment]
I have no idea what you're trying to imply....

**Comment by FibreFoX** [comment]
@ejzhang have you installed gogs as a service? please check the current working path, it probably is not set up correctly.

**Comment by ejzhang** [comment]
@FibreFoX I don't think it should use the current working path, instead of the Gogs installation directory as the default path.
and processing of relative paths in the program is inconsistent, for example, the avatar upload files are alway save in Gogs installation directory but the sqlite3 database file is save in current working direcotry.

**Comment by FibreFoX** [comment]
Just to be sure, have you checked setting the working-dir is helping?

**Comment by ejzhang** [comment]
I can't find the setting of working-dir, How do i check it? thanks very much!
ps: i'm installed the gogs as windows service.

**Comment by FibreFoX** [comment]
How did you install GOGS as windows service? Did you use NSSM or windows built-in service manager?

**Comment by ejzhang** [comment]
I use windows built-in service manager
